oci.go

```diff
@@ -41,6 +41,11 @@ var (
 var cgroupsMemDirPath = "/sys/fs/cgroup"
 
 func containerExists(containerID string) (bool, error) {
+
```

Comment:
Now that I look at it again, nothing seems weird. For some reason I didn't expect to have `false, nil` a valid combination, brain fart I guess.